accepted/0017-add-funding-support.md

```diff
@@ -0,0 +1,138 @@
+# Add `funding` support to `package.json`
+
+## Summary
+
+This RFC identifies an initial means of tooling to describe & notify consumers of a package's monetary support information. `npm` has been [vocal about our commitment](https://blog.npmjs.org/post/187382017885/supporting-open-source-maintainers) to providing a means for package maintainers to more sustainably support their work & this reflects an initial step toward that goal. 
+
+## Motivation
+
+Package maintainers want to clearly indentify how their software is currently, or could be in the future, supported monetarily.
+
+## Detailed Explanation
+
+* See [Prior Art](#prior-art)
+* Provide a means to **reference** existing backers or funding opportunities in `package.json`
+* Provide a means to **define the type** of backing or funding opportunities in `package.json`
+* Provide a means to **notify** package consumers of backers or funding opportunites with the cli
+* Provide a means to **view** dependency's backers or funding opportunites with the cli
+* Provide a means to **view** a package's backers or funding opportunites
+
+##  Rationale & Alternatives
+* Rationale: This is a straightforward and easily implemented solution to take the first step toward supporting Open Source Package Maintainence sustainability
+* Rationale: Building this functionality into the package management layer is the only reliable way to push funding notifications to the vast majority of developers who already use npm
+* Alternative: Continue to rely on third-party, opt-in, tools & platforms that only support a fraction of the desired audience
+* Alternative: Continue debating a larger, more complex schema /w correspondingly more complex tooling
+
+## Implementation
+
+* Add a `funding` field to `package.json`
+  * supports a string URL or object with...
+  * keys that support arbitrary string identifiers
+    * ex. `"sponsor"`, `"sponsors"`, `"donations"`, `"bounties"`, `"contributors"`, `"patrons"` etc.
+  * values that represent a string URL or an array of URLs
+* Add notification at the end of output of package installation that references the number of packages with `funding` defined 
+  * ex. `23 packages are looking for funding. Run "npm fund" to find out more.`
+* Add `--no-fund` flag to opt-out of the funding notification when installing
+* Add `npm fund <pkg>` subcommand: 
+  * if a singular URL is defined for the specified package's `funding` field, it will try to open it using the `--browser` config param (similar to `npm repo <pkg>`)
+  * if an object is defined for the specified package's `funding` field, it will print out the tree of values
```

Comment:
I'd trim this out for now.  Feels like biting off too much at this point, and commits us to supporting whatever arbitrary stuff people put there.  We should be more opinionated about specifically which fields we treat specially, and what each are for.